packages/lms/src/routes/+layout.svelte

```diff
@@ -17,18 +17,17 @@
 <div class="layout-grid">
 	<Logo />
 	<Header>
-		<Search/>
+		<Search />
 		<Settings />
 	</Header>
 	<Molly />
 	<IconNav />
 	<Main>
 		<slot />
 	</Main>
-	<Navigation />
 	<ContentNav />
 	<Footer>
-		<Ego/>
+		<Ego />
 		<Reader />
 	</Footer>
 </div>
```

Comment:
`import Navigation from '$lib/components/navigation/Navigation.svelte';` no longer needed, and it breaks the UI (at least in my local environment)